### PR TITLE
cu: add page

### PR DESCRIPTION
**Pull request by mofosyne**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/cu.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# cu
+
+> **Call Up** another system and act as a dial-in / serial terminal or perform file transfers with no error checking.
```
Comment (spageektti) [inline pages/linux/cu.md:3]:
```suggestion
> Call Up another system and act as a dial-in / serial terminal or perform file transfers with no error checking.
```
We do not support this kind of markdown. See style guide.

File: pages/linux/cu.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open a given serial port with a given baud rate and echo characters locally (half-duplex mode):
+
+`sudo cu --line {{/dev/ttyUSB0}} --speed {{115200}} --halfduplex`
+
+- Open a given serial port with a given baud rate, parity, no hardware or software flow control:
+
+`sudo cu --line {{/dev/ttyUSB0}} --speed {{115200}} --parity={{even|odd|none}} --nortscts --nostop`
+
+- When in a connection exit the `cu` session by typing:
```
Comment (spageektti) [inline pages/linux/cu.md:22]:
```suggestion
- Exit the `cu` session when in connection:
```
I dont know if we support this kind of examples, but it should be in imperative case anyway.

**Comment by spageektti** [comment]
Hi. Thanks fir your contribution. I left you some suggestions. Can you please sign the CLA?

**Comment by mofosyne** [comment]
@spageektti makes sense, I've accepted all the suggestion. CLA signed

**Review by spageektti — APPROVED** [review]
LGTM

**Pull request merged**